projects/robots/bitcraze/crazyflie/controllers/crazyflie/crazyflie.c

```diff
@@ -102,22 +104,25 @@ int main(int argc, char **argv) {
   gains_pid.kp_vel_xy = 2;
   gains_pid.kd_vel_xy = 0.5;
   gains_pid.kp_z = 10;
-  gains_pid.ki_z = 50;
+  gains_pid.ki_z = 5;
   gains_pid.kd_z = 5;
   init_pid_attitude_fixed_height_controller();
 
+  double height_desired = FLYING_ALTITUDE;
+
   // Initialize struct for motor power
   motor_power_t motor_power;
 
-  printf(" Take off! \n");
   printf("\n");
 
   printf("====== Controls =======\n");
 
   printf(" The Crazyflie can be controlled from your keyboard!\n");
   printf(" All controllable movement is in body coordinates\n");
-  printf("- Use the up, back, right and left button to move in the horizontal plane\n");
+  printf("- Use the up, back, right and left button to move in the horizontal "
+         "plane\n");
```

Comment:
This one should be fine for clang-format as we are extending the default line length from 80 to 128 characters.
```suggestion
  printf("- Use the up, back, right and left button to move in the horizontal plane\n");
```

Reply:
Done in [3d59860](https://github.com/cyberbotics/webots/pull/6107/commits/3d598602614f7f338678f46e922b14c7e43326e6)